### PR TITLE
IBX-6264: Clarify what hidden state is

### PR DESCRIPTION
**Pull request by glye**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-6264
| Versions      | all supported
| Edition       | all

Part of a multi-target goal to clarify what "hidden" state is, and warns about wrong usage.

See also:
https://github.com/ezsystems/user-documentation/pull/224
https://github.com/ezsystems/developer-documentation/pull/2092

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [x] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [x] ~Code samples are working~
- [x] ~PHP code samples have been fixed with PHP CS fixer~
- [x] ~Added link to this PR in relevant JIRA ticket or code PR~

### CODE REVIEW

File: docs/content_management/locations.md [hunk 1]
```diff
@@ -94,10 +94,12 @@ When the Location of a Content item is hidden, the system will not display it on
 
 !!! caution "Visibility and permissions"
 
-    The Location visibility flag is not permission-based and thus acts as a simple potential filter.
-    **It is not meant to restrict access to content**.
+    The visibility flag is a convenience feature for withdrawing content from the frontend. It acts as a simple filter in the default frontend templates.
```
Comment (MagdalenaZuba) [inline docs/content_management/locations.md:97]:
```suggestion
    The visibility flag is a convenient feature for withdrawing content from the frontend. It acts as a simple filter in the default frontend templates.
```

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

**Comment by glye** [comment]
@MagdalenaZuba Ok for me to merge now. Btw I have no more than these 3.

**Review by adriendupuis — APPROVED** [review]

**Comment by MagdalenaZuba** [comment]
> 
It acts as a filter in the default frontend templates, you can choose to respect it or ignore it.

**Comment by glye** [comment]
@MagdalenaZuba Hm, maybe clarify who "you" is? Though we are in the dev docs.
"It acts as a filter in the default frontend templates, and developers can choose to respect it or ignore it."

**Comment by MagdalenaZuba** [comment]
@glye I'm ok with it, though in other parts of the documentation when we refer to the reader we use "you".

File: docs/content_management/locations.md [hunk 2]
```diff
@@ -94,10 +94,12 @@ When the Location of a Content item is hidden, the system will not display it on
 
 !!! caution "Visibility and permissions"
 
-    The Location visibility flag is not permission-based and thus acts as a simple potential filter.
-    **It is not meant to restrict access to content**.
+    The visibility flag is a convenient feature for withdrawing content from the frontend. It acts as a simple filter in the default frontend templates.
```
Comment (glye) [inline docs/content_management/locations.md:97]:
```suggestion
    The visibility flag is a convenient feature for withdrawing content from the frontend. It acts as a filter in the frontend by default, but in your code you can choose to respect it or ignore it.
```
@MagdalenaZuba New proposal. Uses "you", makes it clear we're talking about development, not end-user actions, and doesn't specifically mention templates, as the code can be wherever.

Comment (MagdalenaZuba) [reply docs/content_management/locations.md:97]: > @MagdalenaZuba New proposal. Uses "you", makes it clear we're talking about development, not end-user actions, and doesn't specifically mention templates, as the code can be wherever.

```suggestion
    The visibility flag is a convenient feature for withdrawing content from the frontend. It acts as a filter in the frontend by default. You can choose to respect it or ignore it in your code .
```

Comment (glye) [reply docs/content_management/locations.md:97]: Thanks, that's better!

**Comment by glye** [comment]
@MagdalenaZuba If we're cool with this PR now, I guess I should bring the other two in sync with this one. Apart from the links. It's probably good to stay within the dev doc / user doc divide where possible.

**Comment by glye** [comment]
All 3 PRs should be in sync with all reviews now.

**Pull request merged**